lib/procore-sift.rb

```diff
@@ -39,20 +39,31 @@ def filter_errors
   def filter_validator
     @_filter_validator ||= FilterValidator.build(
       filters: filters,
-      sort_fields: self.class.sort_fields,
+      sort_fields: sort_fields,
       filter_params: filter_params,
       sort_params: sort_params,
     )
   end
 
   def filters
-    self.class.filters
+    self.class.ancestors.flat_map do |klass|
+      klass.try(:filters)
+    end.compact.uniq do |f|
+      [f.param, f.class]
```

Comment:
Seems like some extra stuff snuck its way into this method. Logic looks duplicated with half of it not being used?